save-backend/src/main/kotlin/com/saveourtool/save/backend/service/LnkOrganizationTestSuiteService.kt

```diff
@@ -0,0 +1,79 @@
+@file:Suppress("FILE_NAME_MATCH_CLASS")
```

Comment:
why did you add this suppressing here?

Reply:
Used to have a typo in filename, didn't catch it and added suppressing by mistake...
Long story of my intelligence, will fix it

Reply:
Done